From f88dcbadcb6d9a2a6a546e2ed98cc198c0d01ef4 Mon Sep 17 00:00:00 2001
From: Aniket Limaye <a-limaye@ti.com>
Date: Mon, 22 Aug 2022 12:16:17 +0530
Subject: [PATCH 1/2] arm64: dts: ti: j721e-rtos-memory-map: Moving all C7x
 memory to upper 2 GB

- Making space for kernel to reserve memory for CMA
  https://jira.itg.ti.com/browse/ADASVISION-5035

Signed-off-by: Aniket Limaye <a-limaye@ti.com>
---
 arch/arm64/boot/dts/ti/k3-j721e-rtos-memory-map.dtsi | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/arch/arm64/boot/dts/ti/k3-j721e-rtos-memory-map.dtsi b/arch/arm64/boot/dts/ti/k3-j721e-rtos-memory-map.dtsi
index 3f5b7c4ca..0102dd951 100644
--- a/arch/arm64/boot/dts/ti/k3-j721e-rtos-memory-map.dtsi
+++ b/arch/arm64/boot/dts/ti/k3-j721e-rtos-memory-map.dtsi
@@ -217,22 +217,22 @@
 	};
 	vision_apps_core_heaps_lo: vision-apps-core-heap-memory-lo@d8000000 {
 		compatible = "shared-dma-pool";
-		reg = <0x00 0xd8000000 0x00 0x23000000>;
+		reg = <0x00 0xd8000000 0x00 0x0c000000>;
 		no-map;
 	};
-	vision_apps_main_r5fss0_core0_shared_memory_queue_region: vision-apps-r5f-virtual-eth-queues@fb000000 {
+	vision_apps_main_r5fss0_core0_shared_memory_queue_region: vision-apps-r5f-virtual-eth-queues@e4000000 {
 		compatible = "shared-dma-pool";
-		reg = <0x00 0xfb000000 0x00 0x00800000>;
+		reg = <0x00 0xe4000000 0x00 0x00800000>;
 		no-map;
 	};
-	vision_apps_main_r5fss0_core0_shared_memory_bufpool_region: vision-apps-r5f-virtual-eth-buffers@fb800000 {
+	vision_apps_main_r5fss0_core0_shared_memory_bufpool_region: vision-apps-r5f-virtual-eth-buffers@e4800000 {
 		compatible = "shared-dma-pool";
-		reg = <0x00 0xfb800000 0x00 0x01800000>;
+		reg = <0x00 0xe4800000 0x00 0x01800000>;
 		no-map;
 	};
 	vision_apps_core_heaps_hi: vision-apps-core-heap-memory-hi@880000000 {
 		compatible = "shared-dma-pool";
-		reg = <0x08 0x80000000 0x00 0x10000000>;
+		reg = <0x08 0x80000000 0x00 0x27000000>;
 		no-map;
 	};
 };
-- 
2.17.1

